app/assets/stylesheets/app/calendar.scss

```diff
@@ -18,10 +18,32 @@
       line-height: 1;
     }
 
+    tbody {
+      tr:nth-of-type(odd) {
+        .bg-primary {
+          --bs-table-bg-type: $primary;
+        }
+
+        .bg-danger {
+          --bs-table-bg-type: $danger;
+        }
+
+        .bg-warning {
+          --bs-table-bg-type: $warning;
+        }
+      }
+    }
+
```

Comment:
Overrides the global styles for use in the calendar

Reply:
and nicely nested inside `simple-calendar`, lgtm